Runtime/Scripts/LocationService.cs

```diff
@@ -141,7 +146,12 @@ public IEnumerator LocationServiceFlow()
                     yield return StartCoroutine(InitalizeLocationService());
                 }
             }
-            else //iOS - Permission Request are enabled once the application request location info
+            //iOS - Permission Request  once the application request location info
+            if (Application.platform == RuntimePlatform.IPhonePlayer)
+            {
+                yield return StartCoroutine(InitalizeLocationService());
+            }
+            else
             {
                 yield return StartCoroutine(InitalizeLocationService());
             }
```

Comment:
Since it's the same code in both blocks, you can just add a single else block after the `if (Application.platform == RuntimePlatform.Android)`. Or if you think it may be different in the future, you can use a switch and falling through to the default